src/tcas/src/lib/TCasConstants.ts

```diff
@@ -0,0 +1,664 @@
+/* eslint-disable camelcase */
+const REFRESH_RATE: number = 1000; // Time between refreshes in ms
+const TRACKING_MAX: number = 40; // # max contacts tracked - AMM 34-43-00:6a
+const MEMORY_MAX: number = 200; // Max history before culling
+const MIN_VS: number = -6000;
+const MAX_VS: number = 6000;
+const INHIBIT_CLB_RA: number = 39000; // for all climb RA's
+const INHIBIT_INC_DES_RA_AGL: number = 1450; // for increase descent RA's
+const INHIBIT_ALL_DES_RA_AGL: number = 1200; // 1200 takeoff, 1000 approach
+const INHIBIT_ALL_RA: number = 1000; // 1100 in climb, 900 in descent
```

Comment:
I don't understand this. Is there functionality that still needs to be implemented? Why is a constant with 1000 commented with 1100 and 900?

In fact, neither of these two variables seems to be used.

Reply:
There is still functionality that is yet to be implemented. This can be addressed in future PRs.